tests/test_filesystem.py

```diff
@@ -89,6 +89,12 @@ def test_win_devices_not_win(self):
         assert filesystem.sanitize_filename("$mft") == "$mft"
         assert filesystem.sanitize_filename("a$mft") == "a$mft"
 
+    @set_platform("win32")
+    def test_file_illegal_chars_win32(self):
+        assert filesystem.sanitize_filename("test" + filesystem.CH_ILLEGAL_WIN + "aftertest") == (
+            "test" + filesystem.CH_LEGAL_WIN + "aftertest"
```

Comment:
This is not really a good test, since it relies on the constant. 
So a mistake change in the constant would not result in the test failing..
No need to manually test all 31 chars, maybe just 5 or so.

And we also need a test for `sanitize_foldername`.